Dockerfiles/Cassandra-3/cassandra-env.sh

```diff
@@ -231,14 +231,20 @@ fi
 # To enable remote JMX connections, uncomment lines below
 # with authentication and/or ssl enabled. See https://wiki.apache.org/cassandra/JmxSecurity
 #
+LOCAL_JMX=not-only
+
 # Specifies the default port over which Cassandra will be available for
 # JMX connections.
 # For security reasons, you should not expose this port to the internet.  Firewall it if needed.
 JMX_PORT="7199"
-# Make JMX Public
-JVM_OPTS="$JVM_OPTS -Dcassandra.jmx.remote.port=$JMX_PORT"
-JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.rmi.port=$JMX_PORT"
-JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=false"
+
+if [ "$LOCAL_JMX" = "yes" ]; then
+  JVM_OPTS="$JVM_OPTS -Dcassandra.jmx.local.port=$JMX_PORT"
```

Comment:
isn't `DisableExplicitGC` missing here?

Reply:
Not really, since it was changed in Cassandra 3.6: https://issues.apache.org/jira/browse/CASSANDRA-10091

Reply:
Very well.  I had a look at the 3.? docs, it was there, that's why I've asked.

Reply:
You mean datastax docs?  This is stil 3.0, for example: https://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/secureJmxAuthentication.html

Reply:
I mean exactly this page, yes.  The hidden rant is about the f*d-up Cassandra versions.